src/lit-html.ts

```diff
@@ -109,13 +109,11 @@ export function render(
  * An expression marker with embedded unique key to avoid
  * https://github.com/PolymerLabs/lit-html/issues/62
  */
-const attributeMarker = `{{lit-${Math.random()}}}`;
-
-const textMarkerContent = '_-lit-html-_';
-const textMarker = `<!--${textMarkerContent}-->`;
-const attrOrTextRegex = new RegExp(`${attributeMarker}|${textMarker}`);
-const lastAttributeNameRegex =
-    /((?:\w|[.\-_$])+)=(?:[^"']*|(?:["][^"]*)|(?:['][^']*))$/;
+const marker = `{{lit-${String(Math.random()).slice(2)}}}`;
+const nodeMarker = `<!--${marker}-->`;
+const markerRegex = new RegExp(`${marker}|${nodeMarker}`);
+const nonWhitespace = /[^\s]/;
+const lastAttributeNameRegex = /([\w\-.$]+)=(?:[^"']*|"[^"]*|'[^']*)$/;
```

Comment:
Just cleans up the groups.

Reply:
If you think of a way to eliminate this Regex too, then 👍

Reply:
I tried. We could do it if we used `<!--marker-->` for both attr and node markers, but then I hit an issue with unquoted attributes ending the node. I was saving that for the next PR. 😉